site/docs/master/release.md

```diff
@@ -0,0 +1,36 @@
+This file documents the list of steps to perform to create a new Antrea
+release. We use `<TAG>` as a placeholder for the release tag (e.g. `v0.1.0`).
+
+ * Create a release branch for each new minor release (e.g `release-0.1`). For
+   bug fixes, use the appropriate release branch.
+
+ * Open a PR against the appropriate release branch with the following commits:
+    1. a commit to update the [CHANGELOG]({{site.gh_repo}}/tree/master/CHANGELOG.md).
+    2. a commit to update [VERSION]({{site.gh_repo}}/tree/master/VERSION) as needed.
+
+ * Generate the deployment manifest for the release:
+   `IMG_NAME=antrea/antrea-ubuntu IMG_TAG=<TAG> ./hack/generate-manifest.sh --mode release > antrea.yml`
+
+ * Make the release on Github with the release branch as the target: copy the
+   relevant section of the [CHANGELOG]({{site.gh_repo}}/CHANGELOG.md) for the release
```

Comment:
@ccbayer this won't work; because the file is symbolically linked from the root .